src/Orchard/Orchard.Framework.csproj

```diff
@@ -419,7 +427,9 @@
     <Compile Include="Services\ClientHostAddressAccessor.cs" />
     <Compile Include="Services\DefaultJsonConverter.cs" />
     <Compile Include="Services\IClientHostAddressAccessor.cs" />
+    <Compile Include="Services\IYamlParser.cs" />
```

Comment:
The Web.config was modified by NuGet. Should I remove the JSON binding redirect entry?